kokoro/config/test/ops_agent/release.gcl

```diff
@@ -0,0 +1,25 @@
+import 'common.gcl' as common
+import '../image_lists.gcl' as image_lists
+
+_parse_distro = lambda tag: {
+  parts = split(tag, '/')
+  ret = {
+    distro = parts[3]
+    arch = parts[4]
+  }
+}.ret
+
+config build = common.ops_agent_test {
+  params {
+    local _distro_arch = _parse_distro(params.environment._LOUHI_TAG_NAME)
+    platforms = image_lists.get(_distro_arch.distro + '_' + _distro_arch.arch, 'invalid_distro').release
+    arch = _distro_arch.arch
+
+    environment {
+      // The release builds run as a different service account.
+      TRANSFERS_BUCKET = 'stackdriver-test-143416-file-transfers'
+      SERVICE_EMAIL =
+          'build-and-test@stackdriver-test-143416.iam.gserviceaccount.com'
```

Comment:
Can we not just include `release/common.gcl` instead of `common.gcl`?

Reply:
The `release/` folder will be deleted in a future PR.